cmd/broker/filter/main.go

```diff
@@ -114,7 +115,7 @@ func main() {
 		logger.Fatal("Error setting up trace publishing", zap.Error(err))
 	}
 
-	reporter := filter.NewStatsReporter()
+	reporter := filter.NewStatsReporter(env.PodName, "filter")
```

Comment:
Pass the container name down as an env var, rather than implicitly equal values in two different locations.

Reply:
Done, thanks!